### ISSUE TITLE
React 0.14

### ISSUE THREAD

**Issue by ronag** [gaearon/redux-devtools#127]
Currently redux-devtools is what is stopping me from upgrading to React 0.14.

```
Warning: React.findDOMNode is deprecated. Please use ReactDOM.findDOMNode from require('react-dom') instead.
```

in LogMonitor.

**Comment by gaearon** [comment]
Is 0.14 released? It does not appear so.

We can't just go ahead and break compatibility with React 0.13 for the sake of a single warning until 0.14 is released as a stable version.

**Comment by gaearon** [comment]
(To clarify: there are a few PRs fixing 0.14 warnings and I'll merge them when it's out, but not before that.)

**Comment by gaearon** [comment]
And note that it's not really stopping you. It looks like an error but it's just a soft warning. Ignore it and you'll be fine.

**Comment by tomatau** [comment]
If you don't like the error because of OCD (like me) or whatever... I've made a temporary fork here:

package.json
```json
"devDependencies": {
  "redux-devtools": "git+ssh://git@github.com:tomatao/redux-devtools.git",,
```

Simply uses ReactDOM for findDOMNode instead of react.

I'll be deleting the fork when fixes merged.

**Comment by lraivio** [comment]
Thanks @tomatau !

**Comment by tomatau** [comment]
NP! Word of warning... heroku can't npm install from github... so have published this to npm as `redux-devtools-14`... which makes me feel VERY DIRTY.

Any chance a fix can be merged for this instead as it's such a tiny fix.

src/react/LogMonitor.js
```js
import React, { PropTypes, findDOMNode, Component } from 'react';
```

becomes:
```js
import React, { PropTypes, Component } from 'react';
import { findDOMNode } from 'react-dom';
```

I would make the PR here but tests wouldn't run locally for me and don't have time to debug why.

**Comment by tomatau** [comment]
Got the tests running locally (bad npm version) and made a tiny PR
https://github.com/gaearon/redux-devtools/pull/135

**Comment by gaearon** [comment]
It's a tiny fix but it breaks 0.13 compatibility.
